content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,233 @@
+   * - Create purchase order
+     - Yes
+     - :ref:`Yes* <inventory/product_management/po>`
+     - No
+   * - Can be manufactured or subcontracted
+     - :ref:`Yes* <inventory/product_management/manufacture>`
+     - :ref:`Yes* <inventory/product_management/manufacture>`
+     - No
+   * - Can be in a kit
+     - Yes
+     - Yes
+     - No
+   * - Placed in package
+     - Yes
+     - :ref:`Yes* <inventory/product_management/package>`
+     - No
+   * - Appears on Inventory report
+     - :ref:`Yes <inventory/product_management/report>`
+     - No
+     - No
+
+.. _inventory/product_management/on-hand-store:
+
+On-hand quantity
+----------------
+
+A storable product's on-hand and forecasted quantities, based on incoming and outgoing orders, are
+reflected on the product form, accessed by going to :menuselection:`Inventory app --> Products -->
+Products`.
+
+.. figure:: type/on-hand.png
+   :align: center
+   :alt: Show "On Hand" and "Forecast" smart buttons.
+
+   Current and forecasted quantities displayed in the **On Hand** and **Forecasted** smart buttons
+   on the product form.
+
+.. _inventory/product_management/on-hand-con:
+
+On the other hand, consumable products are regarded as always available, and they **cannot** be
+managed using reordering rules or lot/serial numbers.
+
+.. _inventory/product_management/transfer-store:
+
+Create transfer
+---------------
+
+*Transfers* are any warehouse operation, such as receipts, internal or batch transfers, or
+deliveries.
+
+When creating a transfer for storable products in the *Inventory* app, transfers modify the on-hand
+quantity at each location.
+
+For example, transferring five units from the internal location `WH/Stock` to `WH/Packing Zone`
+decreases the recorded quantity at `WH/Stock` and increases it at `WH/Packing Zone`.
+
+.. _inventory/product_management/transfer-con:
+
+For consumable products, transfers can be created, but exact quantities at each storage location are
+not tracked.
+
+.. _inventory/product_management/transfer-serv:
+
+Service products cannot be included in transfers, but these products can be `linked to projects and
+tasks for deadline tracking <https://www.youtube.com/watch?v=fix2LGkv13c>`_.
+
+.. _inventory/product_management/po:
+
+Create purchase order
+---------------------
+
+Both storable and consumable products can be included in a request for quotation in the *Purchase*
+app.
+
+However, when receiving consumable products, their on-hand quantity will not change upon validating
+the receipt (e.g. `WH/IN`).
+
+.. _inventory/product_management/manufacture:
+
+Manufacture or subcontract
+--------------------------
+
+Storable and consumable products can both be manufactured, subcontracted, or included in a bill of
+materials (BoM).
+
+.. figure:: type/manufacture.png
+   :align: center
+   :alt: Show "Bill of Materials" and "Used In" smart buttons.
+
+   When the **Bill of Materials** and **Used In** smart buttons are visible on the product form,
```

Comment:
I'd throw a GUI on both of those smart button mentions because they appear on the UI -- 👍

Reply:
I agree that these deserve a GUI, but the caption text doesn't render like the rest of the doc.

When I use guilabel, no bold shows up
![image](https://github.com/odoo/documentation/assets/23611711/167b3b35-0173-40d6-ae38-c6d513089470)
![image](https://github.com/odoo/documentation/assets/23611711/f2d5b3ad-c5d0-4273-aee8-5e59275446cc)

However, when I use bold, just a littleee bit of bold shows up
![image](https://github.com/odoo/documentation/assets/23611711/379a198c-794e-4839-8d62-4673c3d6d3b2)

so even though it's wrong by our guidelines, to make the terms appear bolded in the result, I'm planning to not use guilabels in the captions.

What do you think about this?